graphql_compiler/query_formatting/common.py

```diff
@@ -26,6 +86,9 @@ def ensure_arguments_are_provided(expected_types, arguments):
         raise GraphQLInvalidArgumentError(u'Missing or unexpected arguments found: '
                                           u'missing {}, unexpected '
                                           u'{}'.format(missing_args, unexpected_args))
+    if check_types:
```

Comment:
Let's remove the option to skip type checking, and always type check. We can consider adding this short-circuiting option back if benchmarks show it to be necessary, but I suspect the query execution time will always dominate the time it takes to run `isinstance` on a few arguments.